src/SIL.XForge.Scripture/ClientApp/src/app/translate/editor/editor.component.ts

```diff
@@ -516,8 +522,8 @@ export class EditorComponent extends DataLoadingComponent implements OnDestroy,
         if (this.projectDoc == null || this.projectDoc.data == null) {
           return;
         }
-        await this.loadNoteThreadDocs(this.projectDoc.id);
         this.text = this.projectDoc.data.texts.find(t => t.bookNum === bookNum);
+        await this.loadNoteThreadDocs(this.projectDoc.id);
```

Comment:
I moved `this.text = ...` one line higher so it will be set before the note threads are loaded. As far as I can tell this wouldn't have any adverse effects.